src/main/java/com/zaiats/catcher/model/CandidateModel.java

```diff
@@ -2,14 +2,27 @@
 
 import com.zaiats.catcher.repository.entity.Candidate;
 
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
 import java.util.Objects;
 
 public class CandidateModel {
 
     private Long id;
+
+    @NotBlank
+    @Size(max = 64)
     private String firstName;
+
+    @NotBlank
+    @Size(max = 64)
     private String lastName;
```

Comment:
I would also like to add a minimum value here

Reply:
1. Same as prev.
2. Default validation message is pretty self-descriptive here.

Reply:
1. The same as previous.
2. Added specific messages to all validations.